src/app/models/SiteSetting.ts

```diff
@@ -0,0 +1,26 @@
+import { Id } from "@interfaces/apiInterfaces";
+import { bawPersistAttr } from "./AttributeDecorators";
+import { AbstractModel } from "./AbstractModel";
+
+export interface ISiteSetting {
+  id?: Id;
+  name: string;
+  value: number;
+  description: string;
+}
+export class SiteSetting
+  extends AbstractModel<ISiteSetting>
+  implements ISiteSetting
+{
+  public readonly kind = "site_setting";
+  public readonly id?: Id;
+  @bawPersistAttr({ update: true })
+  public readonly name: string;
+  @bawPersistAttr({ update: true })
+  public readonly value: number;
+  public readonly description: string;
+
```

Comment:
not all the attributes on this model are exposed here? intentional?

https://github.com/QutEcoacoustics/baw-server/blob/39dc2871dac431c0cc096691f41a55389ca10b29/app/models/admin/site_setting.rb#L58-L65